examples/lorawan/README.md

```diff
@@ -1,38 +1,46 @@
-LoRaWAN
-=======
+## LoRaWAN
 
-Description
------------
+### Description
 
-This application shows a simple use case of LoRaWAN with RIOT.
+This application shows a basic LoRaWAN use-case with RIOT.
 
 By using the real time clock and low-power capabilities of a board, this
-application shows how to program a LoRaWAN Class A device using RIOT.
+application also shows how to program a LoRaWAN Class A device using RIOT.
 
-This application is using the Over-The-Air Activation procedure.
+By default, the application uses the Over-The-Air Activation (OTAA) procedure.
 
-Usage
------
+### Usage
+
+To join a LoRaWAN network using OTAA activation, edit the application
+`Makefile` and set your device information:
+
+    ACTIVATION_MODE ?= otaa
+    DEVEUI ?= 0000000000000000
+    APPEUI ?= 0000000000000000
+    APPKEY ?= 00000000000000000000000000000000
+
+To join a LoRaWAN network using ABP activation, edit the application
+`Makefile` and set your device and LoRaWAN application information:
+
+    ACTIVATION_MODE ?= abp
+    DEVADDR ?= 00000000
+    NWKSKEY ?= 00000000000000000000000000000000
+    APPSKEY ?= 00000000000000000000000000000000
+    RX2_FREQ ?= 869525000
```

Comment:
Can you add a comment for this? you are doing this because of TTN parameters right?